node/core/chain-selection/src/lib.rs

```diff
@@ -398,6 +426,23 @@ where
 							).await?;
 
 							backend.write(write_ops)?;
+
+							if is_validator != Validator::new(leaf.hash, keystore.clone(), &mut sender).await.is_ok() {
+								is_validator = !is_validator;
+								tracing::info!(
+									"👮 Authority status changed to `{}` at block #{}({})",
+									is_validator,
+									leaf.number,
+									leaf.hash
+								);
+							} else {
+								tracing::info!(
+									"🚔 Authority status is `{}` at block #{}({})",
+									is_validator,
+									leaf.number,
+									leaf.hash
+								);
```

Comment:
Spamming the node output on every block isn't a nice idea.

Reply:
Ah, this should have been a trace.

Reply:
Uh, i somehow thought this is only triggered when the status changes - then `info` should be fine as well.

Reply:
This is triggered when it doesn't change so it's spamming as Basti pointed out. If you think it's not useful I'll remove it.

Reply:
got it - else should be `trace`, or removed -  yep.

Reply:
Remove please, it's another trace bloater. The issue we had is really a blue-moon one.

Reply:
done!